cmd/utils/flags.go

```diff
@@ -743,6 +743,11 @@ var (
 		Usage: "External EVM configuration (default = built-in interpreter)",
 		Value: "",
 	}
+	SilkwormFlag = cli.StringFlag{
```

Comment:
let's better move it to `turbo\cli\flags.go`, so we keep this file mostly unchanged and so we can have less issues rebasing new versions of geth

Reply:
done